src/gui/components/props/input/input.jsx

```diff
@@ -1,11 +1,15 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 
+import container from './input.styles'
```

Comment:
We can't call that `container`, we think about component/container.
This is style.

I would rather calling that `import styles from './input.styles'